test/functional/commands/jetpack-compose-e2e-specs.js

```diff
@@ -34,7 +34,7 @@ describe('Jetpack Compose', function () {
     await driver.updateSettings({ driver: 'compose' });
 
     let e = await driver.elementByTagName('lol');
-    await e.text().should.eventually.equal('Click to see dialog');
+    e.isDisplayed().should.eventually.be.true;
```

Comment:
is that expected now that no text will be returned for this element?

Reply:
This node was having the text from its child due to merging, it won't have text in unmerged tree, the same text will now be available on actual child node

Reply:
if an element does not contain text property, It will throw InvalidElementStateException. there was a bug that caused it to throw a different exception

Reply:
Please add missing await, since `eventually` also resolves to a promise